addons/hr_holidays/tests/test_avatar_card_tour.py

```diff
@@ -0,0 +1,185 @@
+from datetime import date, timedelta
+
+from odoo import Command
+from odoo.tests import HttpCase, tagged, users
+from odoo.tests.common import new_test_user
+
+
+@tagged("post_install", "-at_install")
+class TestAvatarCardTour(HttpCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        company_2 = cls.env["res.company"].create(
+            {
+                "name": "Test Company Avatar Card Tour",
+                "country_id": cls.env.ref("base.be").id,
+            }
+        )
+        cls.other_company = company_2
+        new_test_user(
+            cls.env,
+            login="normal_user",
+            company_ids=[Command.link(cls.env.company.id), Command.link(company_2.id)],
+            groups="hr.group_hr_user",
+        )
+        admin_user = cls.env.ref("base.user_admin")
+        admin_user.write({
+            "company_ids": [Command.link(cls.env.company.id), Command.link(company_2.id)],
+        })
+        leave_type = (
+            cls.env["hr.leave.type"]
+            .with_company(company_2)
+            .create(
+                {
+                    "name": "Time Off multi company",
+                    "company_id": company_2.id,
+                    "time_type": "leave",
+                    "requires_allocation": False,
+                }
+            )
+        )
+        partner = (
+            cls.env["res.partner"]
+            .with_company(company_2)
+            .create(
+                {
+                    "name": "Test User Partner",
+                    "company_id": company_2.id,
+                    "email": "test@test.com",
+                    "phone": "123456789",
+                }
+            )
+        )
```

Comment:
Extra company/partner/user could already be built-in if using `MailCommon`, this would make this setup much shorter and to the point (specific to leave stuff).